### DISCUSSION TITLE
Feature Request: Select folder for new notes

### DISCUSSION THREAD

**Discussion by judisohn** [scambier/obsidian-omnisearch#355] [Ideas]
I'm typically using Omnisearch from my Daily Note which is in a dedicated folder. When I create new notes I absolutely do not want it to be in that folder. But that's exactly what the button does. I end up having to move it and then apply the correct template.

I'd love if when I clicked "Create Note" it asked me the folder I wanted to put it (I have a Quick Add shortcut now). At the minimum, it would be great to have a folder selector in Settings under the toggle for "Show Create Note button" that let us select a folder and maybe even select a template if I'm not being too picky. I have Templater installed too. 

Thanks for considering this and a great plug in I use a lot!

**Comment by scambier** [comment]
Added the relevant issue 👍 https://github.com/scambier/obsidian-omnisearch/issues/356
